boards/adafruit/qt_py_esp32s3/doc/index.rst

```diff
@@ -0,0 +1,289 @@
+.. zephyr:board:: adafruit_qt_py_esp32s3
+
+Overview
+********
+
+An Adafruit based Xiao compatible board based on the ESP32-S3, which is great
+for IoT projects and prototyping with new sensors.
+
+For more details see the `Adafruit QT Py ESP32S3`_ product page.
+
+.. figure:: img/adafruit_qt_py_esp32s3.webp
```

Comment:
I think you can remove this figure entry from here. It shall be automatically added. @kartben

Reply:
Looking at some of the other examples, I think you are correct. I have removed it.

Reply:
You may actually see the docs for your PR here :)

https://builds.zephyrproject.io/zephyr/pr/81857/docs/boards/adafruit/qt_py_esp32s3/doc/index.html
https://builds.zephyrproject.io/zephyr/pr/81857/docs/boards/index.html#name=Qt